src/main/java/mate/academy/internetshop/dao/jdbc/ItemDaoJdbcImpl.java

```diff
@@ -0,0 +1,151 @@
+package mate.academy.internetshop.dao.jdbc;
+
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import mate.academy.internetshop.dao.ItemDao;
+import mate.academy.internetshop.lib.Dao;
+import mate.academy.internetshop.model.Item;
+import org.apache.log4j.Logger;
+
+@Dao
+public class ItemDaoJdbcImpl extends AbstractDao<Item> implements ItemDao {
+    private static Logger logger = Logger.getLogger(ItemDaoJdbcImpl.class);
+    private static String DB_NAME = "internetshop";
+
+    public ItemDaoJdbcImpl(Connection connection) {
+        super(connection);
+    }
+
+    @Override
+    public Item create(Item item) throws SQLException {
+        Statement statement = null;
+        String query = "INSERT INTO " + DB_NAME
+                + ".items (`name`, `price`) VALUES ( '"
+                + item.getName() + "', " + item.getPrice() + ");";
+        try {
+            statement = connection.createStatement();
+            statement.executeUpdate(query);
```

Comment:
@gkhrshch based on our retrospective today, add generated id to item

Reply:
I am sorry, must have missed what You said at this point, is that id generated by IdGenerator.class, r the one from sql db?